tests/foreman/ui/test_lifecycleenvironment.py

```diff
@@ -287,8 +287,8 @@ def test_positive_custom_user_view_lce(session, test_name):
     lce_name = gen_string('alpha')
     user_login = gen_string('alpha')
     user_password = gen_string('alpha')
-    org = entities.Organization().create()
-    role = entities.Role(name=role_name).create()
+    org = target_sat.api.Organization().create()
```

Comment:
Non-blocking, but does this test need to create an organization, or can it inherit `module_org` as well?

Reply:
It can.  I may come back later remove all that, but not at this point.